src/_nebari/stages/kubernetes_services/template/modules/kubernetes/services/conda-store/variables.tf

```diff
@@ -86,3 +86,21 @@ variable "conda-store-fs" {
     error_message = "Allowed values for input_parameter are \"cephfs\", or \"nfs\"."
   }
 }
+
+variable "worker-overrides" {
+  description = "Worker-specific overrides for conda-store worker pods, including resources and node group selector."
+  type = object({
+    node_selector = optional(string, null)
```

Comment:
```suggestion
```
node_selector does not work well with AWS dues to zone limitations of the EBS volumes on multi-nodes